Packs/HostIo/Integrations/HostIo/hostio.py

```diff
@@ -0,0 +1,213 @@
+    :param client: the base client
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+    try:
+        client.test_module()
+    except DemistoException as e:
+        if 'Forbidden' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return 'ok'
+
+
+def domain_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    """domain command: Returns domain reputation for a list of domains
+
+    :type client: ``Client``
+    :param client: Hostio client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['domain']`` list of domains or a single domain
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains Domains
+
+    :rtype: ``CommandResults``
+    """
+    domains = argToList(args.get('domain'))
+
+    command_results: List[CommandResults] = []
+    for domain in domains:
+        domain_data = client.get_domain_data(domain)
+
+        if domain_data.get('web', {}).get('date'):
+            domain_date_dt = dateparser.parse(domain_data['web']['date'])
+            if domain_date_dt:
+                domain_data['updated_date'] = domain_date_dt.strftime(DATE_FORMAT)
+
+        score = Common.DBotScore.NONE
+        dbot_score = Common.DBotScore(
+            indicator=domain,
+            integration_name='HostIo',
+            indicator_type=DBotScoreType.DOMAIN,
+            score=score
+        )
+
+        domain_standard_context = Common.Domain(
+            domain=domain,
+            updated_date=domain_data.get('updated_date', None),
+            name_servers=domain_data['web'].get('server', None),
+            registrant_name=domain_data['web'].get('title', None),
+            registrant_country=domain_data['web'].get('country', None),
+            registrant_email=domain_data['web'].get('email', None),
+            registrant_phone=domain_data['web'].get('phone', None),
+            dns=domain_data.get('dns', None),
+            dbot_score=dbot_score
+        )
+
+        readable_output = tableToMarkdown('Domain', domain_data)
+
+        command_results.append(CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='HostIo.Domain',
+            outputs_key_field='domain',
+            outputs=domain_data,
+            indicator=domain_standard_context
+        ))
+    return command_results
+
+
+def search_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    field = args.get('field', None)
+    value = args.get('value', None)
+    limit = args.get('limit', 25)
+
+    data = client.get_search_data(field, value, limit, 0)
+
+    domains = data.get('domains', [])
+    total = data.get('total')
+
+    if total == 0:
+        domains = f'No Domains associated with {field}'
```

Comment:
the `domains` variable is expected to be a list in the context.
this string should be used as the human-readable